<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>710</width>
      <height>710</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="LinesColourLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Цвет линий:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="LinesColourcomboBox">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>черный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>белый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>желтый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>зеленый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>синий</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="BackgroundColorLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Цвет фона:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="BackgroundColourcomboBox">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>белый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>черный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>желтый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>зеленый</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Алгоритм:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_cda">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ЦДА</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_brfloat">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Брезенхема с действительными числами</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_brint">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Брезенхема с целыми числами</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_brstep">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Брезенхема с устранением ступенчатости</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_vu">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ву</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_dll">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Библиотечный</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_drawone">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>230</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нарисовать отрезок</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drawmany">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>590</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нарисовать отрезки</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="8,5">
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Х начала:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_xbeg">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Y начала:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_ybeg">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Х конца:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_xend">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Y конца:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_yend">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>551</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="8,5">
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Х центра:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_xc">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Y центра:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_yc">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Угол:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_angle">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Длина:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_len">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_clean">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>760</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить все</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Алгоритмы рисования отрезков</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
